unit/PapyrusFormTest.cpp

```diff
@@ -10,7 +11,11 @@ TEST_CASE("RegisterForSingleUpdate", "[Papyrus][Form]")
   class CustomForm : public MpForm
   {
   public:
-    void Update() override { counter++; }
+    void Update() override
+    {
+      counter++;
+      std::cout << "Updated!\n";
```

Comment:
why is this needed?

Reply:
It's been needed for testing and shouldn't have been there :) I'll remove it today.